axelrod/strategies/cycler.py

```diff
@@ -72,9 +72,11 @@ def __init__(self, cycle="CCD") -> None:
         self.classifier['memory_depth'] = len(cycle) - 1
 
     def strategy(self, opponent: Player) -> Action:
+        actions = {'C': Actions.C, 'D': Actions.D}
```

Comment:
Yes, I think this is a good move, @marcharper has spoken about potentially changing the actions to be another class so things like this give us the flexibility to do so if we end up doing that.